src/panels/lovelace/components/hui-yaml-editor.ts

```diff
@@ -76,7 +98,12 @@ export class HuiYamlEditor extends HTMLElement {
             cm.replaceSelection(spaces);
           },
         },
+        gutters:
+          this._hass && computeRTL(this._hass)
+            ? ["rtl-gutter", "CodeMirror-linenumbers"]
```

Comment:
I don't understand why we need to have a conditional classname? 

Can't we just select them doing `.rtl .CodeMirror-linenumbers` and then not have to have any changes to CodeMirror classes?

Reply:
Because we need more gutters. The CodeMirror-linenumbers is used by codemirror by default. As-is. Can't put rtl in there. The only way to get something in is using this config property and change the order to push the line numbers so they are not obstructed by the scrollbar.

Here's a picture:
![image](https://user-images.githubusercontent.com/37745463/52525367-4467d600-2cb1-11e9-8f0b-87ec56ffd460.png)

Reply:
But why can't you add a padding/margin to the .CodeMirror-linenumbers class when we're in RTL to account for the scrollbar? We just toggle the `rtl` class on `<hui-yaml-editor>` and then can add our own selector based on the `rtl` class.

Reply:
We can and I did it and the code does look a little better but it doesn't properly work since you have to hardcode a width and then when you add more lines it gets cropped:
![image](https://user-images.githubusercontent.com/37745463/52536443-56a24c80-2d63-11e9-9b22-d2d48c0613cb.png)

![image](https://user-images.githubusercontent.com/37745463/52536449-691c8600-2d63-11e9-8075-6534c84ff3f5.png)

That's why they made the gutters.
So I propose to meet half way. See updated code.